artifactory/commands/golang/go.go

```diff
@@ -201,6 +203,32 @@ func (gc *GoCommand) run() error {
 	return err
 }
 
+func getArtifactoryApiUrl(repoName string, details auth.ServiceDetails) (string, error) {
```

Comment:
Let's add a description for this function.
Also, looks like this function requires a unit test.